aten/src/ATen/core/PythonFallbackKernel.cpp

```diff
@@ -0,0 +1,35 @@
+#include <torch/library.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+
+namespace {
+
+void pythonFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+  const auto& schema = op.schema();
+  const auto num_arguments = schema.arguments().size();
+  for (const auto& ivalue : torch::jit::last(*stack, num_arguments)) {
+    if (ivalue.isTensor()) {
+      auto* interpreter = ivalue.unsafeToTensorImpl()->pyobj_interpreter();
+      if (interpreter) {
+        interpreter->dispatch(op, stack);
+        return;
+      }
+    } else if (ivalue.isList()) {
+      for (const auto& nv : ivalue.toListRef()) {
+        if (nv.isTensor()) {
+          auto* interpreter = nv.unsafeToTensorImpl()->pyobj_interpreter();
+          if (interpreter) {
+            interpreter->dispatch(op, stack);
+            return;
```

Comment:
Still reading, but I see that we dispatch using the interpreter of the first valid tensor argument. Is it possible to have (and should we error check) the case where your op is passed two tensors from different interpreters?

Reply:
This case is possible. When it happens, we will subsequently error inside the actual dispatch call because it will attempt to retrieve PyObjects for each of the Tensor inputs, but the interpreter label for some of them will be wrong. Thus, not necessary to test here!